src/syrupy/__init__.py

```diff
@@ -110,9 +119,18 @@ def pytest_sessionfinish(session: Any, exitstatus: int) -> None:
 
 @pytest.fixture
 def snapshot(request: Any) -> "SnapshotAssertion":
+    default_serializer_plugin = request.config.option.default_serializer_plugin
+
+    if default_serializer_plugin:
+        parts = default_serializer_plugin.split(".")
+        module, extension_class_name = ".".join(parts[:-1]), parts[-1:][0]
+        extension_class = getattr(import_module(module), extension_class_name)
```

Comment:
Using importlib feels sort of hacky but I'm not sure of a different way. The import_module API is awkward since it requires the module to be extracted from the path passed to the command line. 

It might be best to move this to the command-line parsing phase and store the extension_class in request.config.option instead.

Reply:
> It might be best to move this to the command-line parsing phase and store the extension_class in request.config.option instead.

Could be a good idea, but not sure how it would be done or the benefit?

Reply:
Just tried it out, seems to work quite well, [example](https://github.com/tophat/syrupy/compare/add-default-extension-option...iamogbz-patch-1)